authorization/src/main/kotlin/com/ritense/authorization/specification/AuthorizationSpecification.kt

```diff
@@ -86,6 +88,10 @@ abstract class AuthorizationSpecification<T : Any>(
                     && relatedEntityAuthorizationRequest.action == permission.action
             }
             .firstOrNull { permission ->
+                permission.appliesInContext(
```

Comment:
Does this actually do anything? It doesn't seem to be returned or throw anything.

Reply:
The `isAuthorizedForRelatedEntity` method returns true if a permission matches, and false if none matched.

Reply:
But doesn't it do the match on the last statematch in the `firstOrNull` block?